find_broken_msan.py

```diff
@@ -0,0 +1,105 @@
+import csv
+import os
+import multiprocessing
+import subprocess
+import tempfile
+import zipfile
+
+
+OSS_FUZZ_DIR = os.path.dirname(__file__)
+PROJECTS_DIR = os.path.join(OSS_FUZZ_DIR, 'projects')
+MSG_LINES = [
+    '# Disabled MSAN because of https://github.com/google/oss-fuzz/issues/6180',
+    '#  - memory',
+]
+
+def get_msan_projects():
+  msan_projects = []
+  for project in os.listdir(PROJECTS_DIR):
+    project_yaml_path = os.path.join(PROJECTS_DIR, project, 'project.yaml')
+    if not os.path.exists(project_yaml_path):
+      continue
+    with open(project_yaml_path) as fp:
+      project_yaml = fp.read()
+    if 'memory' in project_yaml:
```

Comment:
might just be one minor false positive https://github.com/google/oss-fuzz/blob/30f3a8f1c0f5b072e77d5bea82709db04c53453d/projects/freeimage/project.yaml#L10

Reply:
oh did you manually track this one?

Reply:
I think I wont commit this script. Just wanted to include it so I wouldn't lose track of it.